exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,261 @@
+{
+    "exercise" : "yacht",
+    "version" : "1.0.0",
+    "comments" : [
+        "The dice are represented always as a list of exactly five integers",
+        "with values beteen 1 and 6 inclusive. The category is an integer.",
```

Comment:
> The category is an integer

I found that this makes the JSON file unnecessarily hard to read. A human reader of this file will have to constantly refer to the top of this file to see what category a given number refers to. A machine reader will have the mapping built in, but a human reader must first read this file to build the mapping.

If this JSON file instead used strings such as `"Ones"` or `"LittleStraight"` for categories, the advantage is that it would be more readable for humans. Any potential disadvantages we should be aware of?